pages/api/button.md

```diff
@@ -24,7 +24,7 @@ import Button from '@material-ui/core/Button';
 | <span class="prop-name">color</span> | <span class="prop-type">enum:&nbsp;'default'&nbsp;&#124;<br>&nbsp;'inherit'&nbsp;&#124;<br>&nbsp;'primary'&nbsp;&#124;<br>&nbsp;'secondary'<br> | <span class="prop-default">'default'</span> | The color of the component. It supports those theme colors that make sense for this component. |
 | <span class="prop-name">component</span> | <span class="prop-type">union:&nbsp;string&nbsp;&#124;<br>&nbsp;func&nbsp;&#124;<br>&nbsp;object<br> | <span class="prop-default">'button'</span> | The component used for the root node. Either a string to use a DOM element or a component. |
 | <span class="prop-name">disabled</span> | <span class="prop-type">bool | <span class="prop-default">false</span> | If `true`, the button will be disabled. |
-| <span class="prop-name">disableFocusRipple</span> | <span class="prop-type">bool | <span class="prop-default">false</span> | If `true`, the  keyboard focus ripple will be disabled. `disableRipple` must also be true. |
+| ~~<span class="prop-name">disableFocusRipple</span>~~ | <span class="prop-type">bool | <span class="prop-default">true</span> | *Deprecated*. Focus ripple will be removed because it does not follow Material design.<br><br>If `true`, the  keyboard focus ripple will be disabled. `disableRipple` must also be true. |
```

Comment:
Perhaps I'm being over sensitive, but it isn't that it doesn't follow Material Design, so much that the spec has changed since this component was written. (For the worse IMHO. And not saying I'm a big fan of the focus ripple, but the subtle shade change is not enough to distinguish which component is focused).

Reply:
And `text`. 

Put a `text` button next to an `outlined` button, and focus the `text` button. To the uninitiated, which one is focused? The focused `text` button looks like a (lightly) `filled` button, with neither focused.

While the focus ripple might be unattractive, it's visually distinctive, and draws attention to the focused element (much as the native browser highlight does). I guess in the google employee churn, that critical requirement got lost, and the v2 focus state is the result.

Reply:
Not really a fair comparison to put an outlined button next to a text button. I will work on it again and see how it looks without opacity.